apps/events/models/event.py

```diff
@@ -0,0 +1,47 @@
+from django.db import models
+
+from .constants import (
```

Comment:
Will be great to have full import statements.

```suggestion
from apps.events.models.constants import (
```

Reply:
Okay